cmd/status.go

```diff
@@ -165,18 +167,24 @@ func waitForAlligators(ctx context.Context, kubeClient *k8s.Client, helmRelease
 
 	runningAlligators := 0
 
-	isAlligatorRunningFunc := func() (bool, error) {
+	isAlligatorRunningFunc := func() error {
```

Comment:
same here,  define it outside this func

Reply:
i need the func on same scope to set runningAlligator variable, spinner.Poll except func() error